src/application-intl/ApplicationIntlContext.jsx

```diff
@@ -1,9 +0,0 @@
-import { createContext } from 'react';
-import { intlShape } from 'react-intl';
-
-const ApplicationIntlContext = createContext();
```

Comment:
Does react-intl export their actual Context value, or just the useIntl custom hook? If it's just the hook, we might want to keep this around for the straggler Class-based components that can't use the hook.